packages/calcite-components/src/components/loader/loader.tsx

```diff
@@ -104,7 +104,6 @@ export class Loader implements LocalizedComponent {
                 [CSS.loaderPart]: true,
                 [CSS.loaderPartId(index)]: true,
               }}
-              key={index}
```

Comment:
`key={index}` seems useless?

Lumina has a warning about using index as a key (it suggests not using the key at all in those cases)

Reply:
On closer inspection, I think `key` isn't needed here since the indices won't vary between renders.